packages/k2-alpine/src/theme/tokens/colors.ts

```diff
@@ -0,0 +1,82 @@
+const black = {
+  neutral900: '#1E1E24', // Black Russian
+  neutral850: '#28282E' // Jaguar
+}
+
+const white = {
+  neutral: '#FFFFFF', // White
+  neutral50: '#F2F2F3' // White Lilac
+}
+
+/* const success = {
+  success: '#3AC489', // Shamrock
+  success2: '#47C4AF' // Puerto Rico
+} */
+
+const glass = {
+  light: 'rgba(255, 255, 255, 0.6)', // #FFFFFF
+  light2: 'rgba(161, 161, 170, 0.25)', // #A1A1AA
+  dark: 'rgba(40, 40, 46, 0.85)', // #28282E
+  dark2: 'rgba(129, 129, 137, 0.6)', // #818189
+  dark3: 'rgba(197, 197, 200, 0.25)' // #C5C5C8
+}
+
+/* const gradient = {
+  green: {
+    step0: '#1CC51D',
+    stepF: '#47C4AF'
+  },
+
+  violetPink: {
+    step0: '#B28DFA',
+    step1: '#F9C1FC',
+    step2: '#E47AC1',
+    stepF: '#AD8BF8'
+  },
+
+  greenBlue: {
+    step0: '#7FFBB3',
+    step1: '#ABE3FB',
+    stepF: '#80E5E7'
+  },
+
+  blueMagenta: {
+    step0: '#7BDDFC',
+    step1: '#E567FF',
+    stepF: '#42C8FE'
+  },
+
+  yellowPink: {
+    step0: '#FBF4B3',
+    step1: '#E47AC1',
+    step2: '#E47AC1',
+    stepF: '#F4B591'
+  },
+
+  bluePink: {
+    step0: '#30BEFE',
+    stepF: '#FF7683'
+  }
+} */
+
+export const darkModeColors = {
+  $primaryBackground: white.neutral,
+  $primaryText: black.neutral850,
+
+  $secondaryBackground: glass.dark3,
+  $secondaryText: white.neutral,
+
+  $tertiaryBackground: black.neutral900,
+  $tertiaryText: white.neutral
+}
```

Comment:
for the current k2, the design team actually required us to move away from using these abstract names (primary, secondary) and instead just use the color names (neutral, red, blue,...). are these names from design?

Reply:
I talked to the design team and it's fine. As long as these are mapped to the names they use. I do have to review them and add more though. This will be evolving.

Reply:
I discussed more with Todd and he will implement something similar to https://www.youtube.com/watch?v=1ONxxlJnvdM to handle the different modes. figma and mobile will use the same color names.

Reply:
the work will be done in https://ava-labs.atlassian.net/browse/CP-9163